src/Messages.jl

```diff
@@ -330,6 +330,14 @@ Does header for `key` match `value` (both case-insensitive)?
 hasheader(m, k::AbstractString, v::AbstractString) =
     field_name_isequal(header(m, k), lowercase(v))
 
+"""
+    headercontains(::Message, key, value) -> Bool
+
+Does the header for `key` (interpreted as comma-separated list) contain `value` (both case-insensitive)?
+"""
+headercontains(m, k::AbstractString, v::AbstractString) =
+    any(field_name_isequal.(strip.(split(header(m, k), ",")), lowercase(v)))
```

Comment:
no need to `lowercase` `v` because `field_name_isequal` already does case insensitive comparison?

Need test cases for this new function.

Reply:
(then the `lowercase v` should probably go from the `hasheader` above, too)